RoboFile.php

```diff
@@ -0,0 +1,140 @@
+ * of this program must display Appropriate Legal Notices, as required under
+ * Section 5 of the GNU Affero General Public License version 3.
+ *
+ * In accordance with Section 7(b) of the GNU Affero General Public License version 3,
+ * these Appropriate Legal Notices must retain the display of the "Powered by
+ * SugarCRM" logo and "Supercharged by SuiteCRM" logo. If the display of the logos is not
+ * reasonably feasible for technical reasons, the Appropriate Legal Notices must
+ * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
+ */
+require_once __DIR__.'/vendor/autoload.php';
+/**
+ * This is project's console commands configuration for Robo task runner.
+ *
+ * @see http://robo.li/
+ */
+class RoboFile extends \Robo\Tasks
+{
+    use \SuiteCRM\Robo\OperatingSystemTrait;
+    use \SuiteCRM\Robo\EnvironmentVariablesTrait;
+    use \SuiteCRM\Robo\SassTrait;
+    // define public methods as commands
+
+    /**
+     * configure environment for testing
+     * @see https://docs.suitecrm.com/developer/appendix-c---automated-testing/#_environment_variables
+     * @param array $opts optional command line arguments
+     */
+    public function configureTests(
+        array $opts = [
+            'database_driver' => '',
+            'database_name' => '',
+            'database_host' => '',
+            'database_user' => '',
+            'database_password' => '',
+            'instance_url' => '',
+            'instance_admin_user' => '',
+            'instance_admin_password' => '',
+            'instance_client_id' => '',
+            'instance_client_secret' => '',
+        ]
+    ) {
+        $this->say('Configure Test Environment');
+
+        // Database
+        $this->askDefaultOptionWhenEmpty('Database Driver eg. MYSQL or MSSQL:', 'MYSQL', $opts['database_driver']);
+        $this->askDefaultOptionWhenEmpty('Database host:', 'localhost', $opts['database_host']);
+        $this->askDefaultOptionWhenEmpty('Database Username:', 'suitecrm_tests', $opts['database_user']);
+        $this->askDefaultOptionWhenEmpty('Database User password:', 'suitecrm_tests', $opts['database_password']);
+        $this->askDefaultOptionWhenEmpty('Database Name:', 'sutiecrm_tests', $opts['database_name']);
+
+        // SuiteCRM Instance
+        $this->askDefaultOptionWhenEmpty('Instance URL:', 'http://localhost', $opts['instance_url']);
+        $this->askDefaultOptionWhenEmpty('Instance Admin Username:', 'admin', $opts['instance_admin_user']);
+        $this->askDefaultOptionWhenEmpty('Instance Admin Password:', 'admin1', $opts['instance_admin_password']);
+        $this->askDefaultOptionWhenEmpty('Instance OAuth2 Client ID:', 'suitecrm_client', $opts['instance_client_id']);
+        $this->askDefaultOptionWhenEmpty('Instance OAuth2 Client Secret:', 'secret', $opts['instance_client_secret']);
+
+        if ($this->isOsWindows()) {
+            $this->say('Windows detected');
```

Comment:
Just a hint. You can use **say** after a task ran successfully.
Example:

`
if ($this->taskExec("rm -rf whateva)->run()->wasSuccessful()) {
    $this->say('Removed whateva');
}
`

Reply:
"whateva"